src/CONST.ts

```diff
@@ -1043,7 +1043,6 @@ const CONST = {
             BOTTOM_DOCKED: 'bottom_docked',
             POPOVER: 'popover',
             RIGHT_DOCKED: 'right_docked',
-            ONBOARDING: 'onboarding',
```

Comment:
Why are we removing this?

Reply:
ahh sorry I was supposed to add a comment about it but forgot.

We are removing this because it is not used anywhere in the app.
While I was researching the code I was looking at modals usage across the app, and found this one. I have consulted with @mateuuszzzzz who in the past worked on Onboarding feature, and he confirmed this is a leftover in the code.

No good place to clean this up, might as well here (I hope).